addon/components/au-modal.js

```diff
@@ -36,6 +38,12 @@ export default class AuModal extends Component {
     return this.args.initialFocus ?? '.au-c-modal__title';
   }
 
+  get additionalElements() {
+    return FOCUS_TRAP_ADDITIONAL_ELEMENTS.filter(
+      (element) => document.querySelector(element) !== null
+    );
+  }
```

Comment:
I would just do that in the module scope since this never changes while the app is running but the getter can potentially run multiple times.

Reply:
Not entirely sure what the module scope is. Is that within the `constructor`? Or even outside the component class?

Reply:
Outside the component class, but nevermind, it doesn't matter that much 😄.

Reply:
@Windvis No problem, it is always a good idea to accustom to good practices 😉

Like this?

![CleanShot 2023-05-22 at 19 48 38@2x](https://github.com/appuniversum/ember-appuniversum/assets/18174827/772a9d12-99c5-4577-829e-f320b141c98f)

Reply:
Yea, something like that, but I realized it might cause issues if the elements aren't there yet when the code is executed. It will also cause issues in apps that use FastBoot since `document` isn't a thing there. So keep it as-is! It was premature optimization anyways. If we notice it is slow we can cache the result of the getter.